README.rst

```diff
@@ -62,12 +62,7 @@ Audio & Video
 Installation
 ============
 
-To install PyMC on your system, follow the instructions on the appropriate installation guide:
-
--  `Installing PyMC on MacOS <https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(MacOS)>`__
--  `Installing PyMC on Linux <https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Linux)>`__
--  `Installing PyMC on Windows <https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Windows)>`__
-
+To install PyMC on your system, follow the instructions on the `installation guide <https://www.pymc.io/projects/docs/en/latest/installation.html>`__.
```

Comment:
I would link to stable docs, not to latest. But unlike the comment/change below, this one should not be blocking

Reply:
Nah, generally I'd agree, but for installation instructions in the past we often had to respond to changes/breaks in dependencies quickly, so I'd rather have people look at the latest up to date version.